starter_code/app.js

```diff
@@ -11,8 +11,36 @@ app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
+
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
-app.listen(3000);
+hbs.registerPartials(__dirname + '/views/partials');
+
+app.get('/beers', (req, res, next) => {
+  punkAPI
+  .getBeers()
+  .then(beers => {
+    console.log(beers)
+    res.render('beers', {beers})
+  })
+  .catch(error => {
+    console.log(error);
+  });
+});
+
+hbs.registerPartials(__dirname + '/views/partials');
+
+app.get('/randombeers', (req, res, next) => {
+  punkAPI
+  .getRandom()
+  .then(beers => {
+    res.render('randombeers', {beers})
```

Comment:
Como o método getRandom retorna um array com uma única cerveja, dá pra pegar o elemento no índice zero e manda-lo pro hbs.
```suggestion
const beer = beers[0]
    res.render('randombeers', beer)
```